code/__defines/mobs.dm

```diff
@@ -322,4 +322,10 @@
 // Surgery Stuff
 #define SURGERY_SUCCESS 2 // Proceed with surgery
 #define SURGERY_FAIL 1 // Autofail surgery
-#define SURGERY_IGNORE 0 // Ignore surgery completely and just attack
\ No newline at end of file
+#define SURGERY_IGNORE 0 // Ignore surgery completely and just attack
+
+// Moisture levels, for Skrell
+#define MOISTURE_MAX 3600
+#define MOISTURE_NORMAL 1800
+#define MOISTURE_LOW 900
+#define MOISTURE_NONE 0
```

Comment:
Thaaank you for the defines! Though MOISTURE_NONE might be unnecessary. You can just use 0 and it will be fine.